src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { forwardRef, ReactNode } from 'react';
+import {
+  Checkbox as ChakraCheckbox,
+  RecipeVariantProps,
+  SwitchCheckedChangeDetails,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { checkboxRecipe } from 'src/themes/shared/slotRecipes/checkbox';
+
+type CheckboxVariantProps = RecipeVariantProps<typeof checkboxRecipe>;
+
+export type CheckboxProps = CheckboxVariantProps & {
+  name: string;
+  value?: string;
+  isDisabled?: boolean;
+  isChecked?: boolean;
+  isInvalid?: boolean;
+  isIndeterminate?: boolean;
```

Comment:
We discussed to align with the chakra approach here and add the breaking change to rename the boolean props (check the migration Readme for details)